src/content/web3/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Web3
+description: What is web3 and why is it important?
```

Comment:
```suggestion
description: An introduction to web3 - the next evolution of the internet - and why it matters.
```
Perhaps something like this? Maybe we touch on how it relates to Ethereum? Or not, I'm not sure.